common/changes/master_2017-01-07-07-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona now has a default way of generating initials and colors ",
```

Comment:
Formatting: "Persona: Updated default method of generating initials and colors."